central/complianceoperator/v2/compliancemanager/manager_impl.go

```diff
@@ -245,10 +279,87 @@ func (m *managerImpl) ProcessRescanRequest(_ context.Context, _ interface{}) err
 
 // DeleteScan processes a request to delete an existing compliance scan configuration.
 // TODO(ROX-19540)
-func (m *managerImpl) DeleteScan(_ context.Context, _ interface{}) error {
+func (m *managerImpl) DeleteScan(ctx context.Context, scanID string) error {
 	// TODO:
 	// 1. Validate config exists in database
 	// 2. Lock config so it cannot be edited/updated
 	// 3. Push request to Sensor
-	panic("implement me")
+
+	// Check if scan configuration already exists.
+	scanConfig, found, err := m.scanSettingDS.GetScanConfiguration(ctx, scanID)
+	if err != nil {
+		log.Error(err)
+		return errors.Wrapf(err, "Unable to delete scan configuration ID %q.", scanID)
+	}
+
+	if !found {
+		return errors.Errorf("Scan configuration ID %q does not exist.", scanID)
```

Comment:
use  errors.Wrapf(errox.NotFound, "Scan configuration ID %q does not exist.", scanID)